### DISCUSSION TITLE
Question regarding Multiblock Meshes and Baffles

### DISCUSSION THREAD

**Discussion by ChinmayU78** [su2code/SU2#1248] [Q&A]
Hi,
I've been recently been unsuccessful in running an inviscid supersonic case in SU2 using infinitely thin surfaces (baffles) and multiblock meshes created in Pointwise in 3D. I was wondering if these features are supported in SU2, as my results with a multiblock mesh so far don't seem to be correct? I've attached a screenshot of my results using a wing with a diamond airfoil, as well as my SU2 config file.
[GitHubQuestion.zip](https://github.com/su2code/SU2/files/6033830/GitHubQuestion.zip)

Thanks,
Chinmay
![image](https://user-images.githubusercontent.com/74565109/108957238-be6c1d00-7636-11eb-956a-c12ea515fc65.png)

**Comment by TobiKattmann** [comment]
Hi @ChinmayU78 ,
it would be best to attach the mesh here as well if possible. Mach 6 is not really where I feel comfortable saying a lot of things about so I am not sure if hypersonic regimes bring their own numerical problems (for inviscid problems). Therefore a first suggestion would be to start with Ma 0.6 or sth else slower and ramp freestream Ma  once that worked.  [See this tutorial](https://su2code.github.io/tutorials/Inviscid_Wedge/) maybe for some resource on supersonic flow in SU2.

SU2 is an unstructured solver and therefore [.su2 format](https://su2code.github.io/docs_v7/Mesh-File/) is also unstructured. I dont know Pointwise but I guess multiblock meshes can be fed to structured solvers which can translate that knowledge about the mesh into speed :) As su2 reads that mesh, does sth with it and also writes output I would guess that the pointwise multiblock mesh is written correctly into the .su2 format (you can also check with the Surface with edges representation in paraview if that is the case).
Let me know if that helps, cheers
tobi
